adapters/infoawarebidder.go

```diff
@@ -78,19 +78,19 @@ func pruneImps(imps []openrtb2.Imp, allowedTypes parsedSupports) (int, []error)
 	for i := 0; i < len(imps); i++ {
 		if !allowedTypes.banner && imps[i].Banner != nil {
 			imps[i].Banner = nil
-			errs = append(errs, &errortypes.BadInput{Message: fmt.Sprintf("request.imp[%d] uses banner, but this bidder doesn't support it", i)})
+			errs = append(errs, &errortypes.Warning{Message: fmt.Sprintf("request.imp[%d] uses banner, but this bidder doesn't support it", i)})
```

Comment:
We may want to add a new warning code for this case where a media type is not supported.